javalib/src/main/scala/java/time/LocalTime.scala

```diff
@@ -0,0 +1,379 @@
+package java.time
+
+import scala.scalajs.js
+
+import java.time.temporal._
+
+final class LocalTime private(hour: Int,
+                              minute: Int,
+                              second: Int,
+                              nano: Int)
+    extends Temporal with TemporalAdjuster with Comparable[LocalTime]
+    with java.io.Serializable with Preconditions {
+
+  import LocalTime._
+  import ChronoField._
+  import ChronoUnit._
+
+  private val totalNanos = nano.toLong + (1000000000L * second) +
+      (60000000000L * minute) + (3600000000000L * hour)
+
+  require(hour >= 0 && hour <= 23, s"Invalid value for hour: $hour")
+  require(minute >= 0 && minute <= 59, s"Invalid value for minute: $minute")
+  require(second >= 0 && second <= 59, s"Invalid value for second: $second")
+  require(nano >= 0 && nano <= 999999999, s"Invalid value for nanoOfSecond $nano")
+
+  def isSupported(field: TemporalField): Boolean = field match {
+    case NANO_OF_SECOND | NANO_OF_DAY | MICRO_OF_SECOND | MICRO_OF_DAY |
+         MILLI_OF_SECOND | MILLI_OF_DAY | SECOND_OF_MINUTE | SECOND_OF_DAY |
+         MINUTE_OF_HOUR | MINUTE_OF_DAY | HOUR_OF_AMPM | CLOCK_HOUR_OF_AMPM |
+         HOUR_OF_DAY | CLOCK_HOUR_OF_DAY | AMPM_OF_DAY => true
```

Comment:
As weird as it might look, the second to fourth line should be indented 4 spaces, not 5.
Also I'd recommend putting `true` on the next line, indented 2 spaces wrt. `case`